### DISCUSSION TITLE
Surface Book no touch screen

### DISCUSSION THREAD

**Discussion by bebizzy** [sebanc/brunch#1616] [Q&A]
Installed ChromeOS on my Surface Book. It works great except the touchscreen. 

Anyone have success getting it back? I don't use it all the time, but occasionally would like to pull the screen and as as a tablet.

**Comment by iulian1992** [comment]
i made mine working and very stable on the surface 4 pro,at the start of brunch press down on the arrow and go to grub config and from there u can choose 5.15 kernel, press enter and them press again for the next page and enable ipts_touchscreen pressing space and after that press enter and follow the rest

**Comment by bebizzy** [comment]
Worked... sort of. Seems like it needs to be calibrated as touches are
registering on a different part of the screen. But we're getting closer!
Thanks!



*  Marv Dorner*
  ***@***.***
  701.214.6271 <+1701.214.6271>
  http://bebizzy.com
